pkg/cli/admin/prune/deployments/prune_test.go

```diff
@@ -61,14 +61,17 @@ func TestPruneTask(t *testing.T) {
 			replicas := []metav1.Object{}
 
 			deploymentConfig := mockDeploymentConfig("a", "deployment-config")
-			deployments = append(deployments, deploymentConfig)
+			deployment := mockDeployment("b", "deployment")
+			deployments = append(deployments, deploymentConfig, deployment)
 
 			replicas = append(replicas, withCreated(withStatus(mockReplicationController("a", "build-1", deploymentConfig), deploymentStatusOption), now))
 			replicas = append(replicas, withCreated(withStatus(mockReplicationController("a", "build-2", deploymentConfig), deploymentStatusOption), old))
 			replicas = append(replicas, withSize(withCreated(withStatus(mockReplicationController("a", "build-3-with-replicas", deploymentConfig), deploymentStatusOption), old), 4))
 			replicas = append(replicas, withCreated(withStatus(mockReplicationController("a", "orphan-build-1", nil), deploymentStatusOption), now))
 			replicas = append(replicas, withCreated(withStatus(mockReplicationController("a", "orphan-build-2", nil), deploymentStatusOption), old))
 			replicas = append(replicas, withSize(withCreated(withStatus(mockReplicationController("a", "orphan-build-3-with-replicas", nil), deploymentStatusOption), old), 4))
+			replicas = append(replicas, mockReplicaSet("b", "rs1", deployment))
```

Comment:
This test works fine with `FilterDeploymentsPredicate`, but you're removing that filter above, why?